lambda-functions/elb-cleanup/config.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+// cfg global configuration across the whole
+// services
+var cfg config
+
+// config describes the available configuration
+// of the running service
+type config struct {
+	Debug          bool
+	Region         string
+	ExpirationDays int `mapstructure:"expiration_days"`
+}
+
+// Validate makes sure that the config makes sense
+func (c *config) Validate() error {
+	return nil
+}
+
+// Set the file name of the configurations file
+func init() {
+	viper.AutomaticEnv()
+	viper.SetEnvPrefix("elb-cleanaup")
+
+	defaults := map[string]interface{}{
+		"debug":           false,
```

Comment:
Maybe create a struct for Defaults

Reply:
The viper setDefault requires value of type `interface` thats why it has been done in the way.